src/main/scala/com/workday/warp/persistence/mysql/WarpSlickImplicits.scala

```diff
@@ -0,0 +1,20 @@
+package com.workday.warp.persistence.mysql
+
+import scala.language.{higherKinds, implicitConversions}
+import slick.ast.BaseTypedType
+import slick.lifted.{Query, Rep}
+
+/**
+  * Created by ruiqi.wang
+  */
+trait WarpSlickImplicits extends ColumnImplicits
+
+trait ColumnImplicits {
```

Comment:
I'm a bit unclear on how these 2 traits fit in, and why there are 2. can we make do with just `ColumnImplicits`? not sure what functionality is being added by `WarpSlickImplicits`

Reply:
Hmm, maybe `ColumnImplicits` isn't descriptive enough (maybe `QueryColumnImplicits`?). `WarpSlickFunctions` should also probably be renamed to something like `WarpSlickSingleColumnExtensions`. If you look at the slick source code, they stick all of their extension methods in one file: https://github.com/slick/slick/blob/master/slick/src/main/scala/slick/lifted/ExtensionMethods.scala 

Eventually, I think we may have other custom dsl like the other dsl classes in `ExtensionMethods`. I was trying to separate the dsl class definitions from the instantiation to keep the file from getting too large/cluttered. But I guess there shouldn't be that many so we can just use `WarpSlickImplicits` and have `WarpMySQLProfile` extend that. 

Alternatively, notice how in the slick-postgres library that they separate their extension support into separate modules based on usage and use a companion object (per module) instead that pulls in the jdbc profile to instantiate their conversions: https://github.com/tminglei/slick-pg/blob/master/core/src/main/scala/com/github/tminglei/slickpg/agg/PgAggFuncSupport.scala. This is sort of what I'm doing in `HasWarpSlickDsl`, but instead of a companion object, I'm mixing that in as a trait in `WarpMySqlProfile`

Not entirely sure what the best approach is.